### PR TITLE
feat: update ros2 plugin default qos value

### PR DESCRIPTION
**Pull request by haijiaoqihao**

### CODE REVIEW

**Comment by zhangyi1357** [comment]
Should changing default values update the relevant documentation?

File: src/plugins/ros2_plugin/ros2_channel_backend.cc [hunk 1]
```diff
@@ -485,28 +485,22 @@ void Ros2ChannelBackend::Publish(runtime::core::channel::MsgWrapper& msg_wrapper
 rclcpp::QoS Ros2ChannelBackend::GetQos(const Options::QosOptions& qos_option) {
   rclcpp::QoS qos(qos_option.depth);
 
-  if (qos_option.history == "keep_last") {
```
Comment (AimRT) [inline src/plugins/ros2_plugin/ros2_channel_backend.cc:488]: check

**Comment by AimRT** [comment]
doc

File: src/plugins/ros2_plugin/ros2_channel_backend.cc [hunk 2]
```diff
@@ -254,7 +254,7 @@ bool Ros2ChannelBackend::Subscribe(
 
     const auto& info = subscribe_wrapper.info;
 
-    rclcpp::QoS qos(10);
```
Comment (AimRT) [inline src/plugins/ros2_plugin/ros2_channel_backend.cc:257]: make it configurable

**Pull request merged**